src/Extensions/TimedScopes/TimedScopeProvider.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+using Microsoft.Omex.Extensions.Abstractions;
+
+namespace Microsoft.Omex.Extensions.Logging.TimedScopes
+{
+	internal class TimedScopeProvider : ITimedScopeProvider
+	{
+		public TimedScopeProvider(IMachineInformation machineInformation, TimedScopeEventSource eventSource)
```

Comment:
>TimedScopeProvider [](start = 9, length = 18)

validate?

Reply:
DI initialization will fail if types not resolved